ister.py

```diff
@@ -562,18 +562,27 @@ def create_account(user, target_dir):
     """
 
     os.makedirs(target_dir + get_user_homedir(user["username"]), exist_ok=True)
-    if user.get("uid"):
-        command = "useradd -U -m -u {0} {1}"\
-            .format(user["uid"], user["username"])
+    if user.get("password"):
+        opts = "-p '{0}'".format(user["password"])
     else:
-        command = "useradd -U -m {}".format(user["username"])
+        opts = "-p ''"
```

Comment:
This will allow users without a password set to login by default which is not something we want to allow. The option should be an empty string and if the behavior is desired an empty string should be used as the password in the template.

Reply:
Ugh sorry, that was how it has been X(. Well must have had a good reason for that and it passed security review so not changing the behavior now.